js-weather/weather.js

```diff
@@ -0,0 +1,48 @@
+//関数リテラルは呼び出し元のコードよりも先に記述する
+const getWeather = (data) => {
+    if (data === null) {
+        //<div id = "result">HTMLElementオブジェクトを生成
+        const result = document.getElementById('result');
+        result.textContent = '天気を取得できませんでした';
+    } else {
+        // 天気と気温を取得し、配列にする
+        const weatherConditions = data.weather[0].description;
+        const temp = data.main.temp;
+        const MaxTemp = data.main.temp_max;
+        const MinTemp = data.main.temp_min;
+        const array = [weatherConditions, temp, MaxTemp, MinTemp];
```

Comment:
（参考）
arrayですと、一体何の配列かわからないです。

* ○○の配列
* 〇〇データの集合体

みたいのがわかる名前がつけれるといいですね。
例えば、APIがdescriptionで文字列を返してくれるので、それを活かしつつ array そのものは、天気情報の集合体といえるので、こんなふうに書いてみたり。

```suggestion
        const weatherConditions = [weatherDescription, currentTemp, maxTemp, minTemp];
```